frontend/stylesheets/main.less

```diff
@@ -2,6 +2,7 @@
 @import "../../node_modules/react-sortable-tree/style.css";
 @import "../../node_modules/flexlayout-react/style/light.css";
 @import "../../node_modules/antd/dist/reset.css";
+@import "../../node_modules/react-tooltip/dist/react-tooltip.css";
```

Comment:
If I am understanding the docs correctly, this is not needed anymore:

> info
If you are using v5.13.0 or newer, you don't need to import the css file manually, the styles will be injected into the page by default.

As we have version `^5.27.1` I woudl say we can remove this

Reply:
But maybe the page loads faster without the auto injection of the lib :man-shrugging:

Reply:
The automatic injection didn't work for me. Also importing the css on the JS side didn't work. However, I think that the css import here is in good company :sweat_smile: